BaragonService/src/test/resources/configs/purgeCache-alternate.yaml

```diff
@@ -0,0 +1,4 @@
+purgeCache:
+  enabledTemplates:
+    - static_routes
```

Comment:
just a note that, where possible, we try to note leak any HubSpot-specific implementation into the oss projects. Not harmful here at all, just something to keep in mind for future

Reply:
Ah, will keep that in mind, sorry about that